cclib/io/cmlwriter.py

```diff
@@ -7,7 +7,10 @@
 
 """A writer for chemical markup language (CML) files."""
 
-import xml.etree.cElementTree as ET
+try:
```

Comment:
Could you add a comment explaining why this is necessary?

Reply:
Added a code comment about deprecation. Please let me know if you have any suggestions, Thanks.

Reply:
If you're up for it, it would actually be better to just check the Python version, instead of catching the exception. I believe the fast version is used whenever possible starting from version 3.3

Reply:
I have seen this to be suggested as a common idiom

https://bugs.python.org/msg365079

Reply:
A couple of thoughts:

* For Python 3, we're only supporting versions > 3.3.
* Python 2 is EOL'd.

Can't we just use `xml.etree.ElementTree`? Python 3 is already using the accelerated version under the hood, and do we really care about performance of Python 2 at this point?

Reply:
Oh, I thought this is because something didn't work.... if that's not the case then yes, please, just use the non-c sytnax.

Reply:
Thanks, changed the import as suggested.